### DISCUSSION TITLE
Workflow tokens

### DISCUSSION THREAD

**Discussion by gtoy1118** [twbs/bootstrap#38803] [Q&A]
I am playing around with a fork of the bootstrap repository, and I was using the workflows. I was wondering what permissions the "GITHUB_TOKEN" Secret contains and what permissions it has(so I can make my own)

**Comment by julien-deramond** [comment]
Hey @gtoy1118 

`GITHUB_TOKEN` exists by default in GitHub, no need to create it (see https://docs.github.com/en/actions/security-guides/automatic-token-authentication).

> At the start of each workflow run, GitHub automatically creates a unique GITHUB_TOKEN secret to use in your workflow.
